ext/standard/php_fopen_wrapper.c

```diff
@@ -352,7 +352,7 @@ static php_stream * php_stream_url_wrap_php(php_stream_wrapper *wrapper, const c
 			return NULL;
 		}
 
-		if (!(stream = php_stream_open_wrapper(p + 10, mode, options, opened_path))) {
+		if (!(stream = php_stream_open_wrapper(p + 10, mode, options, opened_path, context))) {
```

Comment:
You probably want to use the `php_stream_open_wrapper_ex` version, which is declaring the context argument: https://github.com/php/php-src/blob/master/main/php_streams.h#L598